.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -56,7 +56,17 @@
            - [ ] Approve the `Approval - Partner notification` approval stage.
       1. - [ ] `Approval - Release` stage
            - [ ] **SYNC POINT**: Wait for Microsoft build release.
-           - [ ] ⚠️ 8.0: Verify the release tag in the `dnceng/dotnet-security-partners` repository matches theVMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ 8.0: Verify the release tag in [dotnet-security-partners](https://dev.azure.com/dotnet-security-partners/_git/dotnet) repository matches the VMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ If a custom tag was needed for this servicing release, mirror it to public repo.
+                - Verify that Microsoft release tag is publicly available
+                - Obtain the SHA, for which we need to create the custom tag, from `Pre-Release` stage `Get associated pipeline runs` step - follow the link to `dotnet-installer-official-ci run` to find the correct SHA.
+                - Fetch changes in internal and public `installer` repos
```

Comment:
For 8.0, wouldn't this be the dotnet/dotnet repo and not the installer repo?

Reply:
I'm not sure. Perhaps this step is for 6.0/7.0 only. @MichaelSimons @mthalman

Reply:
@MichaelSimons @mthalman do we need this step for 8.0 previews and VMR? What would be the target repo, `dotnet/dotnet` or [dotnet-security-partners](https://dev.azure.com/dotnet-security-partners/_git/dotnet)?

Reply:
I think this is applicable for 8.0  The target repo would be dotnet/dotnet

Reply:
Fixed with https://github.com/dotnet/source-build/pull/3593/commits/f91c1e5cfe4d2ab5e6f2f3cf6906070c0ce67a19